### ISSUE TITLE
Suggestion: Option to show generic constraints in multiple lines

### ISSUE THREAD

**Issue by laicasaane** [wmjordan/Codist#211]
The current presentation is hard to read when the constraint list is large.
I think having an option to break down this list into multiple lines would be great.

![image](https://user-images.githubusercontent.com/1594982/183871412-2287ff34-945e-4eaf-804d-436a5b41331c.png)

Sample code

```cs
public interface IRunable { }

public class Runner<T0, T1, T2, T3, T4, T5, T6, T7, T8>
    : IRunable
    where T0 : IRunable, new()
    where T1 : IRunable, new()
    where T2 : IRunable, new()
    where T3 : IRunable, new()
    where T4 : IRunable, new()
    where T5 : IRunable, new()
    where T6 : IRunable, new()
    where T7 : IRunable, new()
    where T8 : IRunable, new()
{
}
```

**Comment by wmjordan** [comment]
Thank you for posting this request.

I don't like the current representation in VS either.
Once it is overridden, I have to handle all situations in the Quick Info title, not only generic constraints.

**Comment by wmjordan** [comment]
Implemented in version 6.6
